modules/developer_manual/pages/core/unit-testing.adoc

```diff
@@ -252,33 +238,33 @@ psql -c 'ALTER USER oc_autotest0 CREATEDB;'
 To run all tests, run the following command:
 
 ....
-make test-php
+make test-php-unit
```

Comment:
This is `test-php-unit` stuff from PR #661 
It should not be backported, because in 10.0 and 10.1 the command is/was just `test-php`

Reply:
will fix